ci/build_wheel.sh

```diff
@@ -21,18 +20,11 @@ PACKAGE_CUDA_SUFFIX="-${RAPIDS_PY_CUDA_SUFFIX}"
 
 # Patch project metadata files to include the CUDA version suffix and version override.
 pyproject_file="${package_dir}/pyproject.toml"
+init_file="${package_dir}/${package_name}/__init__.py"
 
-sed -i "s/^version = .*/version = \"${version_override}\"/g" ${pyproject_file}
 sed -i "s/name = \"${package_name}\"/name = \"${package_name}${PACKAGE_CUDA_SUFFIX}\"/g" ${pyproject_file}
-
-# For nightlies we want to ensure that we're pulling in alphas as well. The
-# easiest way to do so is to augment the spec with a constraint containing a
-# min alpha version that doesn't affect the version bounds but does allow usage
-# of alpha versions for that dependency without --pre
-alpha_spec=''
-if ! rapids-is-release-build; then
-    alpha_spec=',>=0.0.0a0'
-fi
+sed -i "/__version__/ s/= .*/= ${version}/g" ${init_file}
+sed -i "/__git_commit__/ s/= .*/= \"${commit}\"/g" ${init_file}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏

```suggestion
sed -i "/^__version__/ s/= .*/= ${version}/g" ${init_file}
sed -i "/^__git_commit__/ s/= .*/= \"${commit}\"/g" ${init_file}
```

Rationale: as above.